data-access/mandataris-download.ts

```diff
@@ -0,0 +1,216 @@
+    mandaatTypeFilter = `
+      VALUES ?functieCode { ${idValues} }
+      ?mandaat org:role ?bestuursfunctieCode .
+      ?bestuursfunctieCode mu:uuid ?functieCode.
+    `;
+  }
+
+  const queryString = `
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>
+    PREFIX org: <http://www.w3.org/ns/org#>
+    PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+    SELECT DISTINCT ?mandataris
+    WHERE {
+      ?mandataris a mandaat:Mandataris.
+      ?mandataris org:holds ?mandaat.
+
+      ?bestuursorgaan lmb:heeftBestuursperiode ?bestuurspriode.
+      ?bestuursperiode mu:uuid ${sparqlEscapeString(bestuursperiodeId)}.
+
+      ${bestuursorgaanFilter ?? ''}
+      ${onlyActiveFilter ?? ''}
+      ${persoonFilter ?? ''}
+      ${fractieFilter ?? ''}
+      ${mandaatTypeFilter ?? ''}
+    }
+  `;
+
+  const sparqlResult = await query(queryString);
+
+  return getSparqlResults(sparqlResult).map((res) => res.mandataris.value);
+}
+
+async function getPropertiesOfMandatarissen(
+  mandatarisUris: Array<string>,
+  bestuursorgaanId: string | null,
+  sort: { ascOrDesc: 'ASC' | 'DESC'; filterProperty: string } | null,
+): Promise<Array<any>> {
+  let sortFilter: string | null = null;
+  let bestuursorgaanFilter: string | null = null;
+
+  if (sort) {
+    sortFilter = `
+      ORDER BY ${sort.ascOrDesc}(${sort.filterProperty})
+    `;
+  }
+
+  if (bestuursorgaanId) {
+    bestuursorgaanFilter = `?bestuursorgaan mu:uuid ${sparqlEscapeString(
+      bestuursorgaanId,
+    )}.`;
+  }
+
+  const escapedUriValues = mandatarisUris
+    .map((uri) => sparqlEscapeUri(uri))
+    .join(' ');
+  const queryString = `
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>
+    PREFIX org: <http://www.w3.org/ns/org#>
+    PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+    PREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>
+    PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+    PREFIX regorg: <https://www.w3.org/ns/regorg#>
+
+    SELECT DISTINCT ?mandataris ?mandaatLabel ?savePublicatieStatusLabel ?saveRangorde (GROUP_CONCAT(DISTINCT ?beleidsdomeinLabel; SEPARATOR=", ") AS ?beleidsdomeinen) ?saveStatusLabel ?fName ?saveLName ?start ?saveEinde ?saveFractieLabel ?bestuursorgaanLabel
+    WHERE {
+      VALUES ?mandataris { ${escapedUriValues} }
+      ?mandataris a mandaat:Mandataris.
+      ?mandataris org:holds ?mandaat.
+      ?mandataris mandaat:status ?status.
+      ?mandataris mandaat:start ?start.
+
+      ?status skos:prefLabel ?statusLabel.
+
+      ?mandaat org:role ?bestuursfunctie.
+      ?bestuursfunctie skos:prefLabel ?mandaatLabel.
+
+      ?mandaat ^org:hasPost ?bestuursorgaan.
+      ?bestuursorgaan mandaat:isTijdspecialisatieVan ?bestuursorgaanInTijd. 
+      ${bestuursorgaanFilter ?? ''}
```

Comment:
Bummer that this needs to be done but otherwise it will return the mandatarissen for other organen as wel on the organen mandatarissen page